markdown/events/2022/bytesize-34-dsl2-syntax.md

```diff
@@ -18,8 +18,75 @@ Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
 
 Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
 These will be recorded and made available at <https://nf-co.re>
-It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nf-core.slack.com/channels/bytesize) Slack channel!
```

Comment:
```suggestion
It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
```

`nf-core.slack.com` is the `Near Field Core` slack workspace.
We are `nfcore.slack.com`